modules/rpc-util.js

```diff
@@ -10,7 +10,8 @@ class RpcRequest {
                 if(error) {
                     reject(error);
                 } else {
-                    resolve(body);
+                    var balance = JSON.parse(body);
```

Comment:
use let or const always.
https://tylermcginnis.com/var-let-const/

Also JSON.parse also can throw error. Handle that appropriately.